lib/index.js

```diff
@@ -7,6 +8,7 @@ module.exports = {
   getTelemetry,
   setTelemetry,
   gatherTelemetryForUrl,
+  gatherSingleTelemetryForUrl,
```

Comment:
I think I would prefer to name this `gatherTelemetryForUrl` and then rename the other one as `gatherTelemetryPerModuleForUrl`.

Thoughts?

Reply:
yeah agreed, thats more contextual. As discussed, I 'll open another PR to resolve https://github.com/ember-codemods/ember-codemods-telemetry-helpers/issues/29